test/smoke/src/import-in-cjs.cjs

```diff
@@ -0,0 +1,10 @@
+/**
+ * @license
+ * Copyright 2025 Google Inc.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+const puppeteer = require('puppeteer');
```

Comment:
could this be an installation test?

Reply:
Actually this should be handled by it, we only need to move the version we test against for those test to the min supported version.

Reply:
I would prefer not add a new type of tests if possible.